lighthouse-logger/index.js

```diff
@@ -1,17 +1,59 @@
 /**
- * @license Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
I'm not sure what the policy is here, we haven't historically updated years because we touched the file, but it conceptually makes sense to do `2016-2018` here

@brendankenny you're our usual license police :)